### PR TITLE
Added section about yarn :eyes:

### LINKED ISSUE #10: Initial README content
The README needs more specific content instead of the generic Gatsby boilerplate:

- Info about Bot.land
- Goals of the project
- Stream specific info:
  -- "do not review on stream" verbiage
  -- Github name public warning

### PR DESCRIPTION
**Pull request by Rotinx**
Adds yarn section in README.md #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,17 @@ Looking for more guidance? Full documentation for Gatsby lives [on the website](
 
 - **To dive straight into code samples, head [to our documentation](https://www.gatsbyjs.org/docs/).** In particular, check out the _Guides_, _API Reference_, and _Advanced Tutorials_ sections in the sidebar.
 
+## We use Yarn!
+[Yarn](https://yarnpkg.com/) is a actively open sourced package manager being developed by Facebook.
```
Comment (Rotinx) [inline README.md]: I butchered the grammer :|

Comment (cmgriffing) [reply README.md]: "is an actively "

File: README.md [hunk 2]
```diff
@@ -90,6 +90,17 @@ Looking for more guidance? Full documentation for Gatsby lives [on the website](
...
+
+* **Offline Mode.** If you've installed a package before, then you can install it again without an internet connection.
```
Comment (cmgriffing) [inline README.md]: I think we could remove the marketing stuff and just a leave a link for now.

Opening line above is fine after the edit.

I think from here we just want to make sure that yarn is used in all the commands and maybe we add a "yarn install" step to the docs somewhere else.

**Comment by Rotinx** [comment]
Hows that look?

**Comment by cmgriffing** [comment]
We have some merge conflicts in the README.

I also notice that the commit messages are not following the commitizen syntax. I think thats my fault for messing up the dependencies, but if you pull in the latest you should be able to get commitizen going after npm install.

I'm happy to walk through any of this process (pulling in latest changes, etc) on stream.

**Comment by cmgriffing** [comment]
Have you had a chance to look at my comment above?

**Pull request closed without merging**